app-text/sigil/sigil-0.9.13.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit cmake-utils python-single-r1 readme.gentoo-r1 xdg-utils
+
+MY_PN="Sigil"
```

Comment:
Make it into MY_P (add the version), and rename the tarball to MY_P rather than P. It is surprising when the directory inside it is extracted to a different name than tarball itself.